client/src/components/pages/jobs/Jobs.module.css

```diff
@@ -0,0 +1,11 @@
+.mainContainer {
+    padding: 2rem;
+  }
+    .header {
+    color: black;
+  }
+  .cards {
+    display: flex;
+    flex-wrap: wrap;
+  }
+  
```

Comment:
I would change the css here so that can fit 3 cards and not 2 and that these cards take up all the width

Reply:
what do you prefer?

Reply:
ok i did it